Programming_Books.md

```diff
@@ -95,7 +95,9 @@
 
 - [Statistics Playbook](https://www.manning.com/books/statistics-playbook) : How-to guide for statistical analysis using R.
 
-- [The Creative Programmer](https://www.manning.com/books/the-creative-programmer)) : Book about creative problem solving in programming.
+- [The Creative Programmer](https://www.manning.com/books/the-creative-programmer) : Book about creative problem solving in programming.
```

Comment:
Great start @stjepanjurekovi! However, I wonder how this book stands out from other books on this subject. Consider adding this information to pique users’ interest.